File: Niveau 1/8 – Répétitions conditionnées/3 - Département de pédagogie le « c'est plus, c'est moins ».py
secret = int(input())
tour = 1
proposition = int(input())
while proposition != secret:
    if proposition > secret:
        print("c'est moins")
    else:
        print("c'est plus")
    proposition = int(input())
    tour += 1
print("Nombre d'essais nécessaires :")
print(tour)

File: Niveau 1/8 – Répétitions conditionnées/3 - Département de pédagogie le « c'est plus, c'est moins ».cpp
#include <iostream>
using namespace std;
int main()
{
  int nb, prop, n = 1;
  cin >> nb >> prop;
  while (nb != prop)
  {
    if (nb > prop)
    {
      cout << "c'est plus" << endl;
    }
    else
    {
      cout << "c'est moins" << endl;
    }
    cin >> prop;
    n++;
  }
  cout << "Nombre d'essais nécessaires :" << endl
       << n;
}